util/uploadTemplate.sh

```diff
@@ -0,0 +1,97 @@
+#!/bin/bash
+_error_exit() {
+   echo "$1"
+   exit 1
+}
+
+_info() {
+  echo "INFO: $1"
+}
+
+_help() {
+    local -- _cmd=$(basename "$0")
+
+    cat <<EOF
+
+  Usage: ${_cmd} [OPTION]...
+
+  Copy the AWS ParallelCluster Template in an S3 bucket.
+
+  --bucket <bucket>             Bucket where upload the template.
+  --srcdir <src-dir>            Template folder.
```

Comment:
<src-dir> is not the template folder as the help says

Reply:
fixed